crates/biome_service/src/settings.rs

```diff
@@ -547,6 +547,10 @@ impl From<PartialGraphqlConfiguration> for LanguageSettings<GraphqlLanguage> {
             language_setting.formatter.bracket_spacing = formatter.bracket_spacing;
         }
 
+        if let Some(linter) = graphql.linter {
+            language_setting.linter.enabled = linter.enabled;
```

Comment:
I think we can use this workaround to make the graphql linting opt-in. This is how we deal with the css linter:

```suggestion
            // TODO: change RHS to `linter.enabled` when graphql linting is enabled by default
            language_setting.linter.enabled = Some(linter.enabled.unwrap_or_default());
```